Userland/Libraries/LibJS/Runtime/Error.cpp

```diff
@@ -48,6 +52,33 @@ ThrowCompletionOr<void> Error::install_error_cause(Value options)
     return {};
 }
 
+void Error::populate_stack()
+{
+    AK::StringBuilder stack_string_builder {};
+
+    // Note: We roughly follow Chromes formatting
+    // Note: The errors name and message get prepended by ErrorPrototype::stack
+    // Note: We dont want to capture the global exectution context, so we omit the last frame
+    // FIXME: We seem generate an extra stack-frame when calling an Errors constructor
```

Comment:
```suggestion
    // FIXME: We seem to generate an extra stack-frame when calling an Errors constructor
```

Well, yes - that's how function calls work. This FIXME is not actionable, please reword it.

Reply:
Is the new wording better?

Reply:
Yep :)